### PR TITLE
feat: uds core docs

### PR DESCRIPTION
**Pull request by chance-coleman**
## Description
Integrated docs between `uds` and other repos. `uds-core` is the initial repo to be integrated.

The module repo's need to add a go.mod file to their `docs` directory which is then pulled in by the `uds` repo to add the docs to the hugo site. 

The module repo's need to pr changes before the `uds` repo's pr can be pushed in due to failing CI runs because the main branch of the module repo's doesn't have the go.mod file present.

I attempted copy and paste the doc files from `uds` with minimal changes, only changes would be links needing to reference urls instead of static local files. 

Hopefully the only real changes are the:
* go.mod file
* frontmatter addition to doc files
* syntax changes for links and alert messages in docs

#
Will let @Jessy-Morris take the first crack at reviewing this since this is her domain!
#

# How to review
1.  checkout the [core-docs-import](https://github.com/defenseunicorns/uds/tree/core-docs-import) branch on `uds` 
2.  run `npm run start` 
3.  open `localhost:1313` in the browser and there should be `UDS Core` tab to view the docs in this uds-core branch. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Other (security config, docs update, etc)

## Checklist before merging

- [x] Test, docs, adr added or updated as needed
- [x] [Contributor Guide Steps](https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md)(https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md#submitting-a-pull-request) followed

### CODE REVIEW

**Review by Jessy-Morris — DISMISSED** [review]

File: docs/_index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: UDS Core
+linkTitle: UDS Core
+type: docs
+menu:
+    main:
+        weight: 30
+---
```
Comment (rjferguson21) [inline docs/_index.md:8]:
```suggestion
  main:
      weight: 30
---
```

File: docs/_index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+title: UDS Core
+linkTitle: UDS Core
+type: docs
+menu:
+    main:
+        weight: 30
```
Comment (rjferguson21) [inline docs/_index.md]:
```suggestion
menu:
  main:
    weight: 30
```

Comment (rjferguson21) [reply docs/_index.md]: Minor thing,  but I think we want to abide by our existing yaml spacing which is 2?

**Comment by rjferguson21** [comment]
I think there will be some follow on iteration/restructuring as we figure out what we've got now that we've combined the docs from the two repos but I think this is a great step.

**Review by rjferguson21 — APPROVED** [review]

**Pull request merged**